make sure "precip" values are always integers

Apollon77 opened the pull request:
… because at day-change 0:00 sometimes "--" is delivered by the API
which may break type consistency and produce conflicts in InfluxDB (at
least)

Apollon77 committed:
Merge remote-tracking branch 'dschaedl/master'

Apollon77 committed:
Merge branch 'dschaedl/master'

Apollon77 committed:
make sure "precip" values are always integers

… because at day-change 0:00 sometimes "--" is delivered by the API
which may break type consistency and produce conflicts in InfluxDB (at
least)

GermanBluefox reviewed (commented):
Can you increase the version. I am responsible for this adapter too. And it would be nice just to push it to npm. I am a bit underwater now. :)

Apollon77 committed:
increase version and rework

- increase version number
- rework … NaN != 0 :-) (value from server side could be string)

Apollon77 committed:
fix to have null instead of NaN

GermanBluefox reviewed (commented):
What is precipitation? Can it be float?
